.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules/
-yarn-error.log
\ No newline at end of file
+.yarn/
```

Comment:
This should stay committed, but it should be ignored in the `.npmignore` probably.